OsmAnd/res/values/strings.xml

```diff
@@ -418,7 +417,7 @@
     <string name="show_more">Show more</string>
     <string name="tracks_on_map">Displayed tracks</string>
     <string name="quick_action_show_hide_gpx_tracks">Show/hide GPX tracks</string>
-    <string name="quick_action_show_hide_gpx_tracks_descr">Tapping this action button shows or hides selected GPX tracks on the map</string>
+    <string name="quick_action_show_hide_gpx_tracks_descr">Button showing or hiding selected GPX tracks on the map.</string>
```

Comment:
The idea of a button is to push/tap it. It needs not be explained.

Reply:
Yeah, this overlapped, had already shortened it.